src/store/audio.ts

```diff
@@ -469,6 +489,16 @@ export const audioStore = {
         }
       }
     ),
+    [IMPORT_FROM_FILE]: createUILockAction(
+      async ({ state, dispatch  }) => {
+        const filePath = await window.electron.showImportFileDialog({ title: "Import" });
+        if (filePath) {
+          const body = new TextDecoder('utf-8').decode(await window.electron.readFile({ filePath }));
+          const audioItems = parseTextFile(body, state.charactorInfos);
+          audioItems.forEach(item => dispatch(REGISTER_AUDIO_ITEM, { audioItem: item }))
```

Comment:
dispatchが全て完了してから制御を返すと良さそうです。
.mapとPromise.allを使ってこんな感じに書けると思います。
```suggestion
          return Promise.all(audioItems.map(item => dispatch(REGISTER_AUDIO_ITEM, { audioItem: item })))
```